go.mod

```diff
@@ -72,4 +74,5 @@ replace (
 	// antrea/plugins/octant/go.mod also has this replacement since replace statement in dependencies
 	// were ignored. We need to change antrea/plugins/octant/go.mod if there is any change here.
 	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20210318032909-171b6795a2da
+	k8s.io/kubelet => k8s.io/kubelet v0.21.0
```

Comment:
@ramay1 I just noticed this. What is this for?

Reply:
I added this to have the version at v0.21.0; without it, version is changed to v0.21.1 for k8s.io/api, etc

Reply:
One of the CI jobs is failing (go.mod tidiness check)
I'm struggling a bit to see why this `replace` is necessary. In particular, if I remove it and run `go mod tidy`, it doesn't seem to make any difference...

Reply:
@antoninbas Thanks; I just checked and it is not needed; I had to do this for k8s.io/kubernetes.
Also, I ran "make tidy" as the "Go / Check go.mod tidy test" is failing and there are updates to go.sum and plugins/octant/go.sum
I will remove this and add the go.sum changes and post an update